pages/About/About.js

```diff
@@ -0,0 +1,10 @@
+export default function get(req, res) {
+
+  const title = `About`
+
+  res.render(`About/About`, {
+    [title]:  true,
```

Comment:
By the way, this is an example of [JavaScript computed property names](https://www.javascripttutorial.net/es6/javascript-computed-property/).